docs/03-policies/security.md

```diff
@@ -212,7 +212,7 @@ Developers and themers maintaining local sandbox copies of client sites must als
 Developers and themers working on the site vhost (SSH/shell, file system, database) must also:
 
 * Ensure they follow best practices with respect to SSH keys, passphrases and passphrase caching (see above).
-* SSH, SFTP and SCP access to vhosts should only ever be through our central pull keys SSH key distribution system. Access by password, manually installed SSH key, web based "shell" script, port forwarding to 3rd parties or other methods are forbidden by unless authorized in advance by the CIO.
+* SSH, SFTP and SCP access to vhosts should only ever be managed by designated admin(s). Access by password, manually installed SSH keys (other than by admins), web based "shell" script, port forwarding to 3rd parties or other methods are forbidden by unless authorized in advance by the CIO.
```

Comment:
For clarity and conciseness: replace "_should only ever be managed_" with "_are managed exclusively by_"